src/components/userGroups/userGroup.service.ts

```diff
@@ -115,13 +111,9 @@ interface GroupList extends ResponseType {
 }
 
 export async function getGroupList(): Promise<GroupList> {
-    try {
-        const response = await get('user/role/group');
-        const { result } = response;
-        return { ...response, result: result.sort((a, b) => a.name.localeCompare(b.name)) };
-    } catch (err: any) {
-        return err;
-    }
+    const response = await get('user/role/group')
```

Comment:
what will happen in case of error?

Reply:
this function is called inside Promise.allsettled. The error will be seen when we call the function